### PR TITLE
Fix bfv legacy

### PR DESCRIPTION
**Pull request by xinzweb**

### CODE REVIEW

**Comment by xinzweb** [comment]
@hsyuan and @vraghavan78, please take a look. Thanks.

**Comment by xinzweb** [comment]
Pending build farm to pass before pushing. Thanks.

File: src/test/regress/sql/bfv_legacy.sql [hunk 1]
```diff
@@ -674,7 +674,11 @@ SELECT CASE WHEN abs(reltuples-10000000)/10000000 < 0.05 THEN 'reltuples and num
             ELSE 'reltuples and number of records for table tbl_truncate_load are inconsistent'
        END AS remark
 FROM pg_class WHERE oid='tbl_truncate_load'::regclass;
+
+-- start_ignore
 SELECT reltuples FROM pg_class WHERE oid='tbl_truncate_load'::regclass;
+-- end_ignore
```
Comment (hsyuan) [inline src/test/regress/sql/bfv_legacy.sql]: why not just delete this query?

Comment (xinzweb) [inline src/test/regress/sql/bfv_legacy.sql]: This shows the estimation there. It's good to have extra information, even we passed the test with 5% delta.

**Comment by xinzweb** [comment]
@atris or @hlinnaka, anybody +1? This can help make the test more reliable.

**Comment by hlinnaka** [comment]
Looks good to me.

- Heikki

**Comment by atris** [comment]
Post merge +1 from me

**Pull request merged**